README.md

```diff
@@ -54,7 +53,6 @@ Make sure to obtain the corresponding API_KEY, and set it in the `.hardhat.confi
 
 ## Additional resources
 
-* **LAOS Whitepaper**:  For a comprehensive understanding of the LAOS project, you can review the [whitepaper](https://github.com/freeverseio/laos-whitepaper)
-
-* **LAOS Roadmap**: To explore the future plans and updates for LAOS, visit the [roadmap](https://github.com/freeverseio/laos-roadmap)
+- **LAOS Whitepaper**: For a comprehensive understanding of the LAOS project, you can review the [whitepaper](https://github.com/freeverseio/laos-whitepaper)
```

Comment:
try not to use "you" in formal language: "For a comprehensive understanding of the LAOS project, please review the..."

Reply:
ok